src/Common/Core/Header/AssetCollection.php

```diff
@@ -19,7 +19,7 @@ public function __construct(Minifier $minifier)
 
     public function add(Asset $asset, bool $minify = true): void
     {
-        if ($minify) {
+        if ($minify && strpos($asset->getFile(), 'http') !== 0 && strpos($asset->getFile(), '//') !== 0) {
```

Comment:
Wouldn't it be more logical/readable to reset `$minify` to `false` if the url is external.
Also same remark as above.

Reply:
this has been fixed